new-dti-website/src/app/apply/page.tsx

```diff
@@ -24,19 +33,219 @@ const ApplyHero = () => (
         </p>
         <button
           className="rounded-xl py-3 px-[20px] bg-[#A52424] text-white 
-          font-bold hover:bg-white hover:text-[#A52424] w-fit"
+          font-bold hover:bg-[#D63D3D] w-fit"
         >
           <a href={config.applicationLink}>Apply now</a>
         </button>
       </div>
     </div>
+    <div className="relative">
+      <RedBlob className={'right-[-300px]'} intensity={0.7} />
+    </div>
+  </div>
+);
+
+const ApplyFAQ = () => {
+  const sections = ['General Questions', 'Behavioral Prep', 'Technical Prep'];
+  const [question, setQuestion] = useState(sections[0]);
+
+  const buttons = sections.map((section) => (
+    <button
+      className={`md:rounded-[30px] xs:rounded-[15px] font-bold md:text-[20px] xs:text-[9px] md:py-4 md:px-5 xs:py-[10px] 
+        xs:px-2 border-[3px] border-[#FEFEFE] ${
+          section === question ? 'text-[#0C0404] bg-[#FEFEFE]' : ''
+        }`}
+      onClick={() => setQuestion(section)}
+    >
+      {section}
+    </button>
+  ));
+
+  const sectionHeadingStyle =
```

Comment:
Yes! I think it would be a good idea to start reusing them in a css file or better yet in the Tailwind config file, so we can make our own utilities! This would include font styles, weights, etc that could be used like utilities as such that can be reused so we don't have to touch our css file! [https://tailwindcss.com/docs/configuration](url)

#